src/transforms/bin.js

```diff
@@ -171,7 +171,10 @@ function maybeBin(options) {
   return bin;
 }
 
-function maybeThresholds(thresholds = thresholdScott) {
+function maybeThresholds(thresholds) {
+  if (thresholds === undefined) {
+    return (...args) => Math.min(200, thresholdScott(...args));
```

Comment:
Let’s have a `thresholdDefault` static function (a global, but local to this file) rather than creating a closure each time it is needed. Also, would be possible to pass arts explicitly through to thresholdScott since we know which arguments that function expects.